application/controllers/handler.js

```diff
@@ -305,7 +312,7 @@ module.exports = {
           oldEMail = document.email;
 
         if (decodeURI(req.payload.username) !== oldUsername) {
-          return res(boom.notAcceptable('username could not be changed!'));
+          return res(boom.notAcceptable('Username could not be changed!'));
```

Comment:
"Username könnte nicht geändert werden"? Do you mean "It's not possible to change the username!" ?